server/routes/router.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+const apiRouter = require('./api');
+
+router.use('/api/' + process.env.ACTUAL_API_VERSION, apiRouter);
```

Comment:
Don't you think that we can add this API prefix directly in our App class?

Reply:
No, I don't think so. App will be used not only for REST service starts with `api/v1/` but also mock github API service.